staff/gianluca-bragaglia/js-arrays/fill.js

```diff
@@ -0,0 +1,44 @@
+function fill(){
```

Comment:
I like the fact that you tried to make it work without declaring specific arguments in the function but I think that makes really big and confusing code.
Try something like this:

function fill(arr, value, start, end) {
    var length = arr.length;
    if (end == undefined){
        end = length;
    }
    if(start == undefined){
        start = 0;
    }
    for(var i=start;i<end;i++){
        arr[i]=value;
    }
    return arr;
}

Reply:
thanks! I change it.